### ISSUE TITLE
Use yaml configuration files instead of current custom specification

### ISSUE THREAD

**Issue by heuristicus** [ori-drs/procman_ros#17]
The custom specification could probably be replicated in yaml without issue and would mean that the sheriff_config file would no longer be necessary. Perhaps there isn't much to be gained from this, but a custom parser and file format is not ideal.

**Comment by mcamurri** [comment]
I agree that would be nice, but since the internal parser works fine I would keep it that way.

Migrating to yaml would have the only benefit for new users, who do not have to learn a custom syntax for a config file. It is something we might want to consider in the future, but not now.
